internal/servers/console_proxy_core.go

```
@@ -0,0 +1,124 @@
+func ExtractBearerToken(authValue string) (string, error) {
+	if len(authValue) < 7 || !strings.EqualFold(authValue[:7], "Bearer ") {
+		return "", errors.New("Authorization must use Bearer scheme")
+	}
+	return authValue[7:], nil
+}
+
+// OpenTicket decrypts and verifies a raw ticket JWT and logs the result. Both
+// transports call this after extracting the raw token from their respective auth
+// mechanisms.
+func (c *ConsoleProxyCore) OpenTicket(ctx context.Context, rawTicket string) (*console.Ticket, error) {
+	ticket, err := c.Opener.Open(rawTicket)
```

golangci-lint (2.12.2)

[error] Function `Open->Open` should pass the context parameter

(contextcheck)